docs/cloud/general/ci.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "Elementary CI"
+sidebarTitle: "Elementary CI"
+---
+
+### Coming soon - Elementary Pull request impact analysis!
+
+When making changes to your data project, it can sometimes be hard to fully understand the impact of your changes and be certain that there are no unintended consequences.
+
+Our impact analysis will run on every pull request in your dbt project, so that you can see the downstream impact of your changes.
+You'll also be able to see if any of your dbt tests are failing or your models aren't being built successfully.
+
+<img src="/pics/cloud/pr_impact_example.png" />
+
+Elementary CI automations will help you make changes with confidence and seeing the full picture before merging your pull request.
+
+## Want to learn more?
```

Comment:
Change to "Want to join the beta?"
And change the call to schedule a call or reach out to us on Slack